src/jdk.compiler/share/classes/com/sun/tools/javac/jvm/ClassWriter.java

```diff
@@ -1264,7 +1264,7 @@ void writeStackMap(Code code) {
             Assert.checkNull(code.stackMapBuffer);
             for (int i=0; i<nframes; i++) {
                 if (debugstackmap) System.out.print("  " + i + ":");
-                StackMapTableEntry frame = code.stackMapTableBuffer[i];
+                StackMapTableFrame frame = code.stackMapTableBuffer[i];
                 frame.write(this);
```

Comment:
```java
code.stackMapTableBuffer[i].write(this);
```
This variable is only used once, can it be removed?

Reply:
This patch is made by IDE during renames. The patch here is incomplete; we might have a complete patch when value objects jep has its pull request, I think.